### PR TITLE
Add OSD (MSP Displayport) option to port peripherals

### PR DESCRIPTION
**Pull request by SteveCEvans**
This PR compliments https://github.com/betaflight/betaflight/pull/11913 which removes `displayport_msp_serial` and instead uses the first port with the FUNCTION_MSP_DISPLAYPORT flag set, for connection to a suitable VTX.

This PR adds an extra port peripheral option thus:

image

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by haslinghuis** [comment]
Something wrong with workflow environment, lately have to manually restart runs to build correctly. Seems we are on throttle.

**Review by ctzsnooze — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Pull request merged**
